### PR TITLE
feat: candidate Feature File for UCD-754 Save and Continue Navigation

### PR DESCRIPTION
**Pull request by karunak21**
## Ticket Number

UCD-754

## Description of change
Candidate Feature File for Appeal Submission Save and Continue Navigation

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] I have merged commits to avoid fixing things in this PR
- [X ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@wip
```
Comment (frank-dodion) [inline e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature:1]: Do we want the @UI-ONLY tag here?

File: e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    Scenario Outline: A prospective appellant is on a sub-section that is the last part of a section
+        Given a prospective appellant is on the last sub-section <sub-section> of <section>
+        When "Save & Continue" is selected
+        Then the user is taken to the Appeal Submission Tasklist
+        Examples:
+            | section                                 | sub-section                  |
+            | About you                               | About you                    |
+            | About the original planning application | Upload the decision letter   |
+            | About your appeal                       | other-appeals                |
```
Comment (frank-dodion) [inline e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature:46]: Should it be "Other appeals" in the sub-section column?

Comment (karunak21) [reply e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature:46]: Sorry for the typo, it should be 'Other relevant appeals'

File: e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature (hunk 1 above)
Comment (karunak21) [reply e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation.feature:1]: updated

**Review by frank-dodion — APPROVED** [review]

**Pull request merged**